DEVELOPING_OPAL/validate/src/test/scala/org/opalj/fpcf/ThreadRelatedCallsTest.scala

```diff
@@ -0,0 +1,66 @@
+/* BSD 2-Clause License - see OPAL/LICENSE for details. */
+package org.opalj.fpcf
```

Comment:
Fix package declaration (we use `org.opal` on the first line, then one line for each subpackage)